services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -113,7 +113,7 @@ async def _apply_observation_cycle(
 
 
 @dataclass
-class DynamicSidecarsScheduler:
+class DynamicSidecarsScheduler:  # pylint:disable=too-many-instance-attributes
```

Comment:
pylint is already warning you that this class is becoming a monolitic

- DnamicSidecarsScheduler monolitic! Separate background tasks logic. Separation of concerns
- rename module from ``dynamic_sidecar.scheduler.task`` to something that suggests core functionality (it has more than one task!)